src/main/java/htsjdk/samtools/util/StringUtil.java

```diff
@@ -545,4 +545,56 @@ public static int levenshteinDistance(final String string1, final String string2
 
         return i;
     }
+
+    /**
+     * Calculates the hamming distance between two strings s1 and s2.
```

Comment:
Could you add a brief explanation of what the hamming distance is to the javadoc.  I.e. "Calculates the Hamming distance (number of character mismatches) between two strings"